### PR TITLE
GA: Final samples README.md, how to guide for Linux, how to guide for Windows

### PR DESCRIPTION
**Pull request by momuno**
Espp how to guide is not included.  separate PR for that coming.

### CODE REVIEW

File: sdk/samples/iot/README.md [hunk 1]
```diff
@@ -578,33 +749,45 @@ The following samples use SAS authentication to connect to Azure IoT Hub or Azur
...
-      cmake --build .
-      ```
 
-2. From within the `build` directory, run the sample:
+2. Compile and run the sample from within the `build` directory.
 
     Linux:
 
     ```bash
+    make
```
Comment (ericwol-msft) [inline sdk/samples/iot/README.md:778]: Not "cmake --build ." ?

File: sdk/samples/iot/docs/how_to_iot_hub_samples_windows.md [hunk 2]
```diff
@@ -28,234 +34,215 @@ For Windows, the command line examples are based on PowerShell. While Windows  d
...
-7. Create and open the solution for the Azure Embedded SDK for C.
+    ```powershell
+    PS C:\azure-sdk-for-c\sdk\samples\iot> cd ../../..
+    PS C:\azure-sdk-for-c> mkdir build
+    PS C:\azure-sdk-for-c> cd build
+    PS C:\azure-sdk-for-c\build> cmake -DTRANSPORT_PAHO=ON ..
+    ```
 
-    From the root of the cloned repository:
+ 1. Compile and run your sample of choice from within the `build` directory:
```
Comment (ericwol-msft) [inline sdk/samples/iot/docs/how_to_iot_hub_samples_windows.md:199]: Compile and run your sample projects in Visual Studio

**Review by ericwol-msft — APPROVED** [review]

File: sdk/samples/iot/README.md (hunk 1 above)
Comment (momuno) [reply sdk/samples/iot/README.md:778]: Both work.  I can use that if you prefer.

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 3]
```diff
@@ -1,31 +1,49 @@
...
-   
 
-What is covered:
 
-- Configuring Arduino IDE to compile a sample using Azure Embedded SDK for C
-- Configuring and running an IoT Hub client telemetry sample.
-- The sample uses device keys to automatically generate a SAS token for authentication (which is valid by default for one hour).
 
-_The following was run on an Ubuntu Desktop 20.04 environment, with Arduino IDE 1.8.12 and Esp8266 module 2.7.4._
+
```
Comment (ahsonkhan) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:43]: Remove extra new line.
```suggestion
```

File: sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md [hunk 4]
```diff
@@ -1,257 +1,242 @@
-# How to Setup and Run Azure SDK for Embedded C IoT Hub Samples on Linux
+# How to Setup and Run Azure SDK for Embedded C IoT Hub Certificate Samples on Linux
+
+- [How to Setup and Run Azure SDK for Embedded C IoT Hub Certificate Samples on Linux](#how-to-setup-and-run-azure-sdk-for-embedded-c-iot-hub-certificate-samples-on-linux)
```
Comment (ahsonkhan) [inline sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md:3]: Remove this bullet and out-dent the rest?

File: sdk/samples/iot/README.md [hunk 5]
```diff
@@ -578,33 +751,44 @@ The following samples use SAS authentication to connect to Azure IoT Hub or Azur
 
 ## Build and Run the Sample
 
-1. Compile the code:
+1. Build the Azure SDK for Embedded C directory structure.
 
-    - From the sdk root, create a build directory (eg `/build`). Change directory into your build directory.
-    - Build the directory structure and the samples:
+    From the sdk root directory `azure-sdk-for-c`:
```
Comment (ahsonkhan) [inline sdk/samples/iot/README.md:756]: Consider capitalizing `sdk` here and anywhere else we refer to it.
```suggestion
    From the SDK root directory `azure-sdk-for-c`:
```

**Pull request merged**